### DISCUSSION TITLE
OSM Tile on V8

### DISCUSSION THREAD

**Discussion by imakihi** [TerriaJS/terriajs#5079] [Q&A] [answered]
Hi there. I a question regarding raster tile data on V8. 

I recently migrated from V7 to V8. I realized that there is no "url-template" type anymore on the V8. So, I used the "open-street-map" type to show some of my XYZ tiles and I haven't had any luck so far...

I tried to show one of the OSM tile layer like following.
`        {
          "type": "open-street-map",
          "name": "OSM",
          "url": "https://tile.openstreetmap.org/{z}/{x}/{y}.png"
        }`

But I cannot show the OSM layer. I could show the layer (URL) with QGIS though.

Can someone help me to figure out how I can show this layer or other XYZ tile layers?

Thank you!

**Comment by geospatialmapmashup** [comment]
Really removed URL-TEMPLATE I used that alot!
Expecially since API KEYS/TOKENS don't work in Open-Street-Map
for open-street-map Catalog Item you do not include placeholders (at least that's how it is currently in v7 that I am using on the web, mobile and desktop http://earthexplorer.world
open-street-map can't be used for ESRI basemaps since Y/ROW is before the column X

So for your URL
https://tile.openstreetmap.org
leave out {z}/{x}/{y}.png

**Comment by imakihi** [comment, marked as answer]
Thank you for the tip, @geospatialmapmashup!

What I actually wanted to achieve was to show XYZ tiles I created. I store those XYZ tile in my S3 bucket. I tired "open-street-map" since I could make it work with one of Japanese government site URL like following.

`        {
          "type": "open-street-map",
          "name": "Geology Map（産総研地質調査総合センター）",
          "description": "geology map in Japan",
          "url": "https://gbank.gsj.jp/seamless/v2/api/1.2/tiles/{z}/{y}/{x}.png"
        },`

But I had no luck with my own XYZ tile... I used the following URL (I changed this URL but its structure is the same as my tile URL).

`"url": "https://hereismyurl.com/mysite.jp/imagerys/example/{z}/{x}/{y}.png"`

Hiroo

**Reply by imakihi** [reply]
I tried different ways and I finally made it work! I used following syntax to show my local XYZ tile.

Original URL is
"url": "https://s3-somewhere..amazonaws.com/3d/imagery_tiles/tokyo_23ku/{z}/{x}/{y}.png"

And I wrote like following and it worked!

`{
          "type": "open-street-map",
          "name": "tokyo-airphoto",
          "url": "https://s3-somewhere.amazonaws.com/3d/imagery_tiles/tokyo_23ku"
},`

I just removed /{z}/{x}/{y}.png from my URL.

I just wanted to share my finding.
